src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml

```diff
@@ -105,8 +105,7 @@
                              Text="{Binding Path=SearchText, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
                              MinWidth="200"
                              CaretBrush="{StaticResource CommonSidebarTextColor}"
-                             Margin="26,0,0,-1"
-                             TextChanged="OnSearchTextBoxTextChanged" />
+                             Margin="26,0,0,-1"/>
```

Comment:
We already bind the Text control to the "SearchText" Path.
When the text is changed, the "SearchText" property's setter will be called...and that is when we will execute the Search command

Reply:
could this also be fixed by making this a one way binding?

Reply:
Changing around the binding types broke other parts of the UI (ex. making it one way did not update the Textbox initial value "Search" anymore)